### DISCUSSION TITLE
Convert httpYac to OpenAPI

### DISCUSSION THREAD

**Discussion by andiKolb** [AnWeber/vscode-httpyac#288] [Q&A]
Is it possible to convert http files from httpYac to an OpenAPI .json file or .yml file? I need this file type for some automated testing purposes that only support this.

**Comment by AnWeber** [comment]
No, there is no converter for this. I'm also not sure how useful it is. The API can only determine the possible request and response based on the transferred data. This will usually be incomplete, i.e. the most you can do is create a template that is then further adapted. It is more effective to do the generation on the server side. DTO objects are usually available there that can generate the schema completely.
